pkg/route/api/helper.go

```diff
@@ -20,11 +20,16 @@ func RouteLessThan(route1, route2 *Route) bool {
 	if route1.CreationTimestamp.Before(route2.CreationTimestamp) {
 		return true
 	}
-	if route1.CreationTimestamp == route2.CreationTimestamp && route1.UID < route2.UID {
-		return true
-	}
-	if route1.Namespace < route2.Namespace {
-		return true
+
+	if route1.CreationTimestamp == route2.CreationTimestamp {
+		if route1.UID < route2.UID {
+			return true
+		}
+		if route1.Namespace < route2.Namespace {
```

Comment:
Will do - have to check implications to the other tests (`plugin_tests`). Should be ok I think - just need to ensure that UIDs are not the same in any of tests.